Layout Design for Each row in SettingsActivity

AVI5HEK opened the pull request:
Layout Design for SettingsActivity
Implementation for Edit/Save button
Toggle text when 'Edit/Save' button clicked
Add row when 'Edit' clicked.

ahammad-reyad commented on app/src/main/java/com/framgia/localchat/MainActivity.java:
```diff
@@ -0,0 +1,39 @@
+        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
+        setSupportActionBar(toolbar);
+
+
```
remove extra line

ahammad-reyad commented on app/src/main/java/com/framgia/localchat/MainActivity.java:
```diff
@@ -0,0 +1,39 @@
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
```
move this to initUI method

AVI5HEK replied:
haven't started coding on MainActivity yet.

ahammad-reyad replied:
@AVI5HEK 
Still change

ahammad-reyad commented on app/src/main/java/com/framgia/localchat/SettingsActivity.java:
```diff
@@ -0,0 +1,59 @@
+        }
+        return super.onOptionsItemSelected(item);
+    }
+}
```
Change implementation to RecyclerView

AVI5HEK replied:
I'll do, sensei. (bow)

ahammad-reyad commented on app/src/main/res/layout/content_main.xml:
```diff
@@ -0,0 +1,20 @@
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Hello World!"/>
```
Move to string.xml

shawon-sharma replied:
please put it in String file

AVI5HEK replied:
No, I'll delete this view. :facepunch:

ahammad-reyad commented on app/src/main/res/layout/settings_row.xml:
```diff
@@ -0,0 +1,47 @@
+
+    <ImageView
+        android:id="@+id/image_avatar"
+        android:layout_width="0dp"
```
dimen

shawon-sharma replied:
put the value in dimens

shawon-sharma commented on app/src/main/java/com/framgia/localchat/SettingsActivity.java:
```diff
@@ -0,0 +1,59 @@
+    private void inflateRow(int position) {
+        LayoutInflater inflater = (LayoutInflater) getSystemService(LAYOUT_INFLATER_SERVICE);
+        final View rowView = inflater.inflate(R.layout.settings_row, null);
+        final EditText etName = (EditText) rowView.findViewById(R.id.et_name);
```
please initialize all the widget eg.(textview,button ) in a particular method  and also make an eventListenerManager() to set listener.

AVI5HEK replied:
@shawon-sharma Why, sensei? :baby:

shawon-sharma commented on app/src/main/res/layout/settings_row.xml:
```diff
@@ -0,0 +1,47 @@
+    <CheckBox
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_weight="1"/>
```
please put values in dimens,String folder

AVI5HEK committed:
Layout Design for Each row in SettingsActivity
Layout Design for SettingsActivity
Implementation for Edit/Save button
Toggle text when 'Edit/Save' button clicked
Add row when 'Edit' clicked.
